diff --git a/syntax_checkers/php/php.vim b/../php.vim
index a6abffb..0ba0d44 100644
--- a/syntax_checkers/php/php.vim
+++ b/../php.vim
@@ -27,10 +27,11 @@ function! SyntaxCheckers_php_php_GetHighlightRegex(item)
     return '\V'.split(unexpected, "'")[1]
 endfunction
 
+" using -n in args speeds things up by bypassing any php.ini parsing
 function! SyntaxCheckers_php_php_GetLocList()
     let makeprg = syntastic#makeprg#build({
-        \ 'exe': 'php',
-        \ 'args': '-l -d error_reporting=E_ALL -d display_errors=1 -d log_errors=0 -d xdebug.cli_color=0',
+        \ 'exe': '/usr/bin/php',
+        \ 'args': '-l -n -d error_reporting=E_ALL -d display_errors=1 -d log_errors=0 -d xdebug.cli_color=0',
         \ 'filetype': 'php',
         \ 'subchecker': 'php' })
 
