--- php.vim	2014-01-31 22:10:43.000000000 -0600
+++ php.vim.fix	2014-01-31 22:11:28.000000000 -0600
@@ -26,7 +26,7 @@
 function! SyntaxCheckers_php_php_GetLocList() dict
     let makeprg = self.makeprgBuild({
         \ 'args': '-d error_reporting=E_ALL',
-        \ 'args_after': '-l -d display_errors=1 -d log_errors=0 -d xdebug.cli_color=0' })
+        \ 'args_after': '-l -d -n display_errors=1 -d log_errors=0 -d xdebug.cli_color=0' })
 
     let errorformat =
         \ '%-GNo syntax errors detected in%.%#,'.
