--- php.vim	2013-07-28 13:31:21.000000000 -0500
+++ php.vim.new	2013-08-13 16:50:07.000000000 -0500
@@ -30,7 +30,7 @@
 function! SyntaxCheckers_php_php_GetLocList()
     let makeprg = syntastic#makeprg#build({
         \ 'exe': 'php',
-        \ 'args': '-l -d error_reporting=E_ALL -d display_errors=1 -d log_errors=0 -d xdebug.cli_color=0',
+        \ 'args': '-l -d -n error_reporting=E_ALL -d display_errors=1 -d log_errors=0 -d xdebug.cli_color=0',
         \ 'filetype': 'php',
         \ 'subchecker': 'php' })
 
